File: python/global_state/globalstatetest.py
import unittest
from globalstate import Configuration


class GlobalStateTest(unittest.TestCase):
    def test_get_db_name(self):
        self.assertEqual(Configuration.get_value("DB_NAME"),"dev")

    def test_get_db_user(self):
        self.assertEqual(Configuration.get_value("DB_USER"),"developer")


if __name__ == '__main__':
    unittest.main()

File: javascript/global_state/globalstate.test.ts
import { expect, test } from 'vitest';
// import { Configuration, ConfigurationReadyToTest } from './globalstate';
import * as config from './globalstate';

test('should get DB name', () => {
    expect(config.Configuration.getValue('DB_NAME')).to.equal('dev');
});

test('should get DB user', () => {
    expect(config.Configuration.getValue('DB_USER')).to.equal('developer');
});


